### ISSUE TITLE
(DB/Locale): Search the Bole

### ISSUE THREAD

**Issue by PivanDepolo** [azerothcore/azerothcore-wotlk#14173]
### Current Behaviour

https://www.wowhead.com/quest=26477/search-the-bole

The quest, when the required conditions are met, is not translated into Russian
When submitting and when the quest is in progress, the phrases are also not translated

### Expected Blizzlike Behaviour

When passing the quest there is no translation into Russian

### Steps to reproduce the problem

Just take this quest

### AC rev. hash/commit

AzerothCore rev. 20da358afbd5+

### Operating system

Debian 11

### Custom changes or Modules

![1](https://user-images.githubusercontent.com/64589699/206971435-e9755246-1e53-44f2-b2ed-ae28532d03b4.png)
![2](https://user-images.githubusercontent.com/64589699/206971439-4016432c-9bf0-4412-ad38-8894d1360f8a.png)
![3](https://user-images.githubusercontent.com/64589699/206971441-5644a516-880b-4f88-b6e5-20bdcb5ead84.png)

**Comment by heyitsbench** [comment]
The linked Wowhead quest does not exist in Wrath.

**Comment by acidmanifesto** [comment]
> The linked Wowhead quest does not exist in Wrath.

Correct. This was a quest from cata that seems to be in the db. Even though the wooden key is a vanilla item, it was used for another quest. This fetch quest should not be present in the acdb.

**Comment by heyitsbench** [comment]
I think the issue is that @PivanDepolo accidentally linked the wrong quest in the issue.
